specs/protocol/predeploys.md

```diff
@@ -140,6 +137,8 @@ value is hardcoded to 6.
 
 ## ProtocolVault
```

Comment:
`ValidatorRewardVault` also.
```suggestion
## [DEPRECATED] ProtocolVault
```

Reply:
If remove deprecated L1Block and L1FeeVault, ProtocolVault and ValidatorRewardVault should be also removed